pol-core/pol/scrdef.cpp

```diff
@@ -99,9 +99,14 @@ bool ScriptDef::config_nodie( const std::string& iname, const Plib::Package* ipk
   return true;
 }
 
+std::string ScriptDef::qualifiedname() const
+{
+  return relativename( nullptr );
```

Comment:
I think if pkg_ is null, this will return the unqualified name, rather than `::x/y/z`.

Suggestion to fix:
```suggestion
  if ( name_ == "" )
    return "";
  else
    return ":" + ( pkg_ ? pkg_->name() : "" ) + ":" + localname_.get();
```